CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 1.34.0
+
+- update `only_throw_errors` to not report on values of type `Never`
+- update `prefer_mixin` to handle class mixins
+- update `unnecessary_null_checks` to ignore `Future.value` and 
+  `Completer.complete`
+- fix `unnecessary_parenthesis` false-positives on constant patterns
```

Comment:
nit: we spell this 'false positives' elsewhere.

Reply:
Thanks!